mesheryctl/internal/cli/root/root.go

```diff
@@ -125,6 +126,12 @@ func initConfig() {
 		viper.SetConfigFile(cfgFile)
 	} else {
 		// Use default ".meshery" folder location.
+		if _, err := os.Stat(utils.MesheryFolder); err != nil {
+			if os.IsNotExist(err) {
+				os.MkdirAll(utils.MesheryFolder, 0775)
+			}
```

Comment:
Looks right. There’s a small lint check item that you might give consideration to.

Reply:
pushed the lint fix